package.json

```diff
@@ -81,6 +81,7 @@
     "nodemon": "^1.10.0",
     "sinon": "^1.17.4",
     "supertest": "^3.0.0",
+    "simple-mock": "^0.8.0",
```

Comment:
`sinon` should already do the job for mocks, spies and stubs, no?

Reply:
I did not know what sinon did.  Was just looking at most of the tests that are in there now.  But yea, let me change it to sinon instead

Reply:
So actually, Romain gave me a way of following the patter there by skipping the validation.  I would probably prefer to do it that way to follow the existing pattern rather than introduce a new one.  But we'll see if it holds up

Reply:
nevermind, Romain gave us a way to test and follow the existing pattern, so going with that as discussed